src/anyvar/storage/postgres_init.sql

```diff
@@ -4,3 +4,7 @@ DROP USER anyvar;
 CREATE USER anyvar WITH PASSWORD 'anyvar-pw';
 CREATE DATABASE anyvar;
 GRANT ALL PRIVILEGES ON DATABASE anyvar TO anyvar;
+
+\c anyvar
+GRANT ALL PRIVILEGES ON SCHEMA public TO anyvar;
+ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT ALL PRIVILEGES ON TABLES TO anyvar;
```

Comment:
I had to do this in order to create the `vrs_objects` table. Did anyone else have to add these permissions?